### DISCUSSION TITLE
best clients panel start page

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#5539] [Q&A]
hi

since the latest update of my adguardhome instance (108.0.28), every statitics panel are numbered as Z to A classification.
that suggest me adguard home doesn't filter anything.
to review the good classifiction i have to click on the request  title number
[bug or not bug.docx](https://github.com/AdguardTeam/AdGuardHome/files/10834184/bug.or.not.bug.docx)

you can see what i mean in the attachment
[bug or not bug.docx](https://github.com/AdguardTeam/AdGuardHome/files/10834185/bug.or.not.bug.docx)

how can you reolve this issue ? 

thanks
[bug or not bug.docx](https://github.com/AdguardTeam/AdGuardHome/files/10834187/bug.or.not.bug.docx)

**Comment by ainar-g** [comment]
Dup of #5522?

**Reply by ghost** [reply]
hi 

may be

so if i understand, i have just to wait the next update for ADH
or is there anything i could try ?

let me know

thanks
